aws-http-nio-client-netty/pom.xml

```diff
@@ -50,6 +50,8 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-native-epoll</artifactId>
+            <version>${netty.version}</version>
+            <classifier>linux-x86_64</classifier>
```

Comment:
Hmm but it is much faster and I think the majority of our customers are on Linux/Amazon Linux so I'd much rather opt into the performant approach by default. What's the concern with adding it? It's a pretty small jar, 134KB. I agree that we don't have to deal with every single native implementation in core but I'd be nice to cater to the big one.